Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -2,6 +2,7 @@ import CarPlay
 import MapboxCoreNavigation
 import MapboxDirections
 import MapboxMaps
+import os.log
```

Comment:
We'll need to add logging for `CPTemplate` lifecycle callbacks in `CarPlayManagerDelegate`:
- carPlayManager(_:templateWillAppear:animated:)
- carPlayManager(_:templateDidAppear:animated:)
- carPlayManager(_:templateWillDisappear:animated:)
- carPlayManager(_:templateDidDisappear:animated:)